src/adhocracy_core/adhocracy_core/authentication/__init__.py

```diff
@@ -90,6 +93,46 @@ def wrapped_view(context, request):
     return wrapped_view
 
 
+def is_header_password(request: IRequest) -> bool:
+    """Check if request provided the password in the Password header."""
+    return UserPasswordHeader in request.headers
+
+
+def get_header_password(request: IRequest) -> str:
```

Comment:
I am not sure about the conventions, but I would prefer `get_header_password(request, default)` to mirror the behavior of `Dict.get()`